modules/dnn/src/dnn.cpp

```diff
@@ -2501,6 +2539,268 @@ struct Net::Impl : public detail::NetImplBase
 #endif
     }
 
+#ifdef HAVE_TIMVX
+    // update all comsumer
+    void tvUpdateConfictMap(int graphIndex, LayerData& ld, std::vector<std::vector<int> >& graphConflictMap)
```

Comment:
To resolve conflicts with merged #21662 please:
- move methods declaration into `modules/dnn/src/net_impl.hpp` file + put `timVxInfo` declaration near it (after all common fields)
- methods implementation should be moved into `op_timvx.cpp` (see halide code as example)

Reply:
@alalek Thanks for your guidance. We have completed the code refactoring.